packages/datadog_grpc_interceptor/lib/src/tracing_util.dart

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import 'dart:math';
+
+final random = Random();
+
+String generateTraceId() {
+  final highBits = random.nextInt(1 << 32);
```

Comment:
I'm not quite sure about trace ID format, but this implementation is different from https://github.com/DataDog/browser-sdk/blob/bb0a5780ea2f6ca092262a01524b957fe4f7c966/packages/rum-core/src/domain/tracing/tracer.ts#L127, should we align? wdyt?

Reply:
Trace ID sent to APM is supposed to be a 64-bit unsigned int, converted to decimal (although, I think RUM wants it in hex... I should double check that).

I think it's the same in theory, just not in practice. JS is storing 63-bits in a Uint8Array than working down digit by digit through the combined 32-bit integers to create the string. This is doing the same, I'm just using Dart's BigInt abstraction to do the heavy lifting for me, but if you look at its internals it does exactly that (and allow me to use the full 64-bits as an unsigned int. Dart only supports 64-bit signed through it's native int type).